src/components/AiNotemaker.tsx

```
@@ -0,0 +1,958 @@
+																"group flex gap-3 p-4 bg-white border rounded-2xl transition-all duration-300 cursor-pointer select-none",
+																item.selected
+																	? "border-green-200 shadow-sm hover:shadow-md hover:border-green-300"
+																	: "border-gray-100 opacity-60 hover:opacity-100 hover:border-gray-300"
+															)}
+															key={i}
+															onClick={() => {
+																const newTasks = [...editableTasks];
+																newTasks[i].selected = !newTasks[i].selected;
+																setEditableTasks(newTasks);
+															}}
+														>
```

Biome (2.4.14)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)